vmhost/hostCore/execution.go

```diff
@@ -676,6 +680,9 @@ func (host *vmHost) CreateNewContract(input *vmcommon.ContractCreateInput) (newC
 	if err != nil {
 		return
 	}
+
+	host.Output().CompleteLogEntriesWithCallType("DeployFromSource")
```

Comment:
Add UpgradeFromSource, UpgradeSmartContract and CreateSmartContract logs as well.